test/unit/config_test.rb

```diff
@@ -603,6 +673,63 @@ def fixture(fixture_name)
           }
         }
       EOJ6
+    when :basic_1_2
+      <<~EOJ7
+        {
+          "version": "1.2",
+          "cli_options": {
+            "create_lockfile": "false"
+          },
+          "reporter": {
+            "automate" : {
+              "url": "http://some.where",
+              "token" : "YOUR_A2_ADMIN_TOKEN"
+            }
+          },
+          "credentials": {
+            "ssh": {
+              "set1": {
+                "host": "some.host",
+                "user": "some_user"
+              }
+            }
+          },
+          "plugins": {
+            "inspec-test-plugin": {
+              "test_key_01":"test_value_01"
+            }
+          }
+        }
+      EOJ7
+    when :bad_1_2_array
+      <<~EOJ8
+        {
+          "version": "1.2",
+          "plugins": [
+            "inspec-test-plugin1",
+            "inspec-test-plugin2"
+          ]
+        }
+      EOJ8
+    when :bad_1_2_bad_name
+      <<~EOJ9
+        {
+          "version": "1.2",
+          "plugins": {
+            "spectacles-banana-peeler": {
+            }
+          }
+        }
+      EOJ9
+    when :bad_1_2_bad_value
+      <<~EOJ10
+        {
+          "version": "1.2",
+          "plugins": {
+            "inspec-test-bad-settings": 42
+          }
+        }
+      EOJ10
```

Comment:
these need not have different heredoc terminator symbols...

Reply:
And unless some of them are syntactically incorrect, they need not be strings. Might be easier to maintain if they're actual objects w/ a to_json on the end?

Reply:
I personally want them to be clearly strings, in the interest of pasting them into JSON linters, JSON config files, etc - I don't want them to be POROs.